ts/src/hyperliquid.ts

```diff
@@ -2619,6 +2619,12 @@ export default class hyperliquid extends Exchange {
             'rejected': 'rejected',
             'marginCanceled': 'canceled',
         };
+        if (status !== undefined && status.endsWith('Rejected')) {
```

Comment:
@gaugau3000 thanks for fixing it, but instead of checking twice if status is undefined, can't we do it just once in the top of the method 

```
if (status === undefined) {
    return undefined;
}
```
and then we don't need to cehck it here

Reply:
Should be oki now ^^. Yks !